sqlmesh/core/loader.py

```diff
@@ -195,6 +195,9 @@ def _load_sql_models(
             for path in self._glob_paths(context_path / c.MODELS, config=config, extension=".sql"):
                 self._track_file(path)
 
+                if not os.path.getsize(path):
```

Comment:
[Nit] We actually read the entire file few lines above. We can just check what we read:
```
model_payload = file.read()
if not model_payload:
    continue
```

Reply:
Seems like we only read the file on cache miss so there is no way to skip the iteration easily. 

Please disregard my comment.

Reply:
@izeigerman An empty file, or an invalid model file, wouldn't be cached because _load would've raised a ConfigError, right? Should the loader catch these ConfigErrors and ignore these files and continue to load instead of failing?